<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResPanel</class>
 <widget class="QDockWidget" name="ResPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="m_resDirView">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="m_searchLineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="m_viewTypeButton">
            <property name="toolTip">
             <string>List</string>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../Resource/App.qrc">
              <normaloff>:/icon/Icon/res/view_type_list.png</normaloff>:/icon/Icon/res/view_type_list.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="m_listView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditKeyPressed</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="m_actionNewFolder">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/folder.png</normaloff>:/icon/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="m_actionShowInExplorer">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/res/folder.png</normaloff>:/icon/res/folder.png</iconset>
   </property>
   <property name="text">
    <string>Show In Explorer</string>
   </property>
  </action>
  <action name="m_actionDeleteRes">
   <property name="icon">
    <iconset resource="../../../Resource/App.qrc">
     <normaloff>:/icon/Icon/file/delete.png</normaloff>:/icon/Icon/file/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="m_actionRenameRes">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="m_actionDuplicateRes">
   <property name="text">
    <string>Duplicate</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../Resource/App.qrc"/>
 </resources>
 <connections/>
</ui>
